dbms/src/Storages/AlterCommands.h

```diff
@@ -12,13 +12,19 @@ namespace DB
 
 class ASTAlterCommand;
 
-/// Operation from the ALTER query (except for manipulation with PART/PARTITION). Adding Nested columns is not expanded to add individual columns.
+/// Operation from the ALTER query (except for manipulation with PART/PARTITION).
+/// Adding Nested columns is not expanded to add individual columns.
 struct AlterCommand
 {
     enum Type
     {
         ADD_COLUMN,
         DROP_COLUMN,
+
+        FREEZE_ALL,
+        // Even though this command operates on partitions, it needs global locks to prevent table alteration.
```

Comment:
Let's write comments before the described line.
Because many IDEs and code viewers expect them in that place.